vagrant/config/versions.yaml

```diff
@@ -1,55 +1,11 @@
 installers:
-  - foreman: '3.3'
-    katello: '4.5'
-    pulpcore: '3.18'
-    puppet: 7
-    boxes:
-      - 'almalinux8'
-      - 'centos7'
-      - 'centos7-fips'
-      - 'centos8'
-      - 'centos8-stream'
-      - 'debian10'
-      - 'debian11'
-      - 'ubuntu2004'
-
-  - foreman: '3.4'
-    katello: '4.6'
-    pulpcore: '3.18'
-    puppet: 7
-    boxes:
-      - 'almalinux8'
-      - 'centos8-stream'
-      - 'debian11'
-      - 'ubuntu2004'
 
   - foreman: '3.5'
```

Comment:
This can be removed too -- While we want to keep support for supported Satellite releases (aka: Foreman 3.5) we do not need the definition here at that files is not used downstream.